src/hash_to_curve.rs

```diff
@@ -165,14 +170,152 @@ where
         let mut us = [C::Base::ZERO; 2];
         hash_to_field("SSWU", curve_id, domain_prefix, message, &mut us);
 
-        let [q0, q1]: [C; 2] = us.map(|u| simple_svdw_map_to_curve(u, z));
+        let [q0, q1]: [C; 2] = us.map(|u| simple_svdw_map_to_curve::<C>(u, z));
 
         let r = q0 + &q1;
         debug_assert!(bool::from(r.is_on_curve()));
         r
     })
 }
 
+// Implementation of <https://datatracker.ietf.org/doc/html/rfc9380#name-simplified-swu-for-ab-0>
+#[allow(clippy::type_complexity)]
+pub(crate) fn simple_svdw_hash_to_curve_secp256k1<'a>(
```

Comment:
naming, it should be sswu, svdw refers to something different in the spec

Reply:
Hmm, okay, I think that `sswu` is more accurate name. (It is used in other codes.)

Reply:
@davidnevadoc 
I see that you built those `simple_svdw_...` naming.
Is it okay if I replace all of those with `sswu_...`, in this PR?

Reply:
Sure, I got that wrong, let's rename it everywhere in this PR.

Reply:
Done https://github.com/privacy-scaling-explorations/halo2curves/pull/110/commits/31690de773abda8ba029f173e614e76f40faea9f